docs/getting-started-objective-c.md

```diff
@@ -35,10 +35,10 @@ Objective-C is a language that is most commonly used to write applications for m
 
 ### macOS
 
-[Creating a macOS application](getting-started-objc-macos) is easiest since it does not involve as many additional steps, like setting up identity, provisining profiles, simulators and devices. You are encouraged to start with the macOS document before the one for iOS.
+[Creating a macOS application](getting-started-objc-macos.md) is easiest since it does not involve as many additional steps, like setting up identity, provisining profiles, simulators and devices. You are encouraged to start with the macOS document before the one for iOS.
```

Comment:
are you sure the `.md` extensions are needed ? iirc most of our documents (in master or in xamarin-macios wiki) don't use them.